docs/kpod-stop.1.md

```diff
@@ -0,0 +1,27 @@
+## kpod-stop"1" "July 2017" "kpod"
+
+## NAME
+kpod stop - Stops one or more containers
+
+## SYNOPSIS
+**kpod** **stop** **containerID[...]**
+
+## DESCRIPTION
+Stops one or more containers.
+
+## OPTIONS
+
+**--timeout, -t**
+
+Seconds to wait for stop before stopping the container. The default is 10.
```

Comment:
Maybe I'm misreading the code, but should this be: "Seconds to wait for stop to complete gracefully before raising an error.  The default is 10."  As is it is described now, this kind of sounds like timeout is use to wait between stops.